diff --git a/usb_modeswitch.c b/usb_modeswitch.c
index 52598f7..cbb8c28 100644
--- a/usb_modeswitch.c
+++ b/usb_modeswitch.c
@@ -2060,7 +2060,8 @@ void printHelp()
 	" -g, --device-num NUM          system device number (for hard ID)\n"
 	" -m, --message-endpoint NUM    direct the message transfer there (optional)\n"
 	" -M, --message-content <msg>   message to send (hex number as string)\n"
-	" -2 <msg>, -3 <msg>            additional messages to send (-n recommended)\n"
+	" -2, --message-content2 <msg>  additional messages to send (-n recommended)\n"
+	" -3, --message-content3 <msg>  additional messages to send (-n recommended)\n"
 	" -w, --release-delay NUM       wait NUM ms before releasing the interface\n"
 	" -n, --need-response           obsolete, no effect (always on)\n"
 	" -r, --response-endpoint NUM   read response from there (optional)\n"
@@ -2084,7 +2085,7 @@ void printHelp()
 	" -Q, --quiet                   don't show progress or error messages\n"
 	" -W, --verbose                 print all settings and debug output\n"
 	" -D, --sysmode                 specific result and syslog message\n"
-	" -s, --success <seconds>       switching result check with timeout\n"
+	" -s, --check-success <seconds> switching result check with timeout\n"
 	" -I, --inquire                 obsolete, no effect\n\n"
 	" -c, --config-file <filename>  load long configuration from file\n\n"
 	" -t, --stdinput                read long configuration from stdin\n\n"
